amz_review_scraper/soup_searcher.py

```diff
@@ -0,0 +1,11 @@
+
+def find_attribute(soup, key, html_tag, attrs):
+    for divs in soup.findAll(html_tag, attrs=attrs):
+        if html_tag == "span":
+            return divs.text.strip()
+        else:
+            try:
```

Comment:
now that you have some nice tests in place, you can refactor this method with confidence!

So I feel like this try, except is a little verbose for what you want to accomplish.
Maybe you check if the key is present in the map Ala:
```python
if key in divs:
  return divs[key]
```

or maybe use `get` which doesn't raise an error if not found.

also is `divs` multiple divs in this context? It feels like a little weird to iterate through something, and the variable you are passed in plural.
Meaning usually these statements look like:

`for div in divs:`